examples/dt/bgp/control-plane/nncp/values.yaml

```diff
@@ -189,9 +189,9 @@ data:
         "bridge": "octbr",
         "ipam": {
           "type": "whereabouts",
-          "range": "172.24.0.0/24",
-          "range_start": "172.24.0.30",
-          "range_end": "172.24.0.70"
+          "range": "172.23.0.0/24",
```

Comment:
Can we use user provider CIDR? If no, I would need to change in the downstream configurations that we pass.

Reply:
Thank you @beagles for the clarification.